lib/cypress-rails/init.rb

```diff
@@ -1,30 +1,44 @@
 module CypressRails
   class Init
-    DEFAULT_CONFIG = {
-      "screenshotsFolder" => "tmp/cypress_screenshots",
-      "videosFolder" => "tmp/cypress_videos",
-      "trashAssetsBeforeRuns" => false
-    }
+    DEFAULT_CONFIG = <<~CYPRESS_CONFIG
+    const { defineConfig } = require('cypress')
+
+    module.exports = defineConfig({
+      // setupNodeEvents can be defined in either
+      // the e2e or component configuration
+      e2e: {
+        setupNodeEvents(on, config) {
+          on('before:browser:launch', (browser = {}, launchOptions) => {
+            /* ... */
+          })
+        },
+      },
+      screenshotsFolder: "tmp/cypress_screenshots",
+      videosFolder: "tmp/cypress_videos",
+      trashAssetsBeforeRuns: false
+    })
+
+    CYPRESS_CONFIG
```

Comment:
Recommend reformatting:

```suggestion
    DEFAULT_CONFIG = <<~CYPRESS_CONFIG
      const { defineConfig } = require('cypress')
  
      module.exports = defineConfig({
        // setupNodeEvents can be defined in either
        // the e2e or component configuration
        e2e: {
          setupNodeEvents(on, config) {
            on('before:browser:launch', (browser = {}, launchOptions) => {
              /* ... */
            })
          },
        },
        screenshotsFolder: "tmp/cypress_screenshots",
        videosFolder: "tmp/cypress_videos",
        trashAssetsBeforeRuns: false
      })
    CYPRESS_CONFIG
```

Reply:
These Cypress setting content formatted in this commit.
ref: https://github.com/testdouble/cypress-rails/pull/156/commits/ea4319e412841b528c96904b1c6f32b35a38f503